example/package.json

```diff
@@ -14,7 +14,7 @@
     "verbose": true
   },
   "scripts": {
-    "test": "ava",
+    "test": "ava && for x in `python -c 'import os; import sys; [print(os.path.realpath(path)) for path in sys.argv[1:]]' lambdas/*`; do BASE=$PWD && cd $x && npm run test && cd $BASE; done",
```

Comment:
Because doing it in a subshell fails, and macs don't have realpath installed by default. 

Or.... are you asking why have the python testing as part of the test command?

Reply:
tl;dr the reason behind it is I didn't know about 'cd -' 

7535bff7e